src/main/resources/languages/english.xml

```diff
@@ -2194,4 +2194,37 @@
 		<key>Converter_EdgeDetection.sampleSize</key>
 		<value>SampleSize</value>
 	</string>
+
+	<string>
+		<key>Generator_FlowField.name</key>
+		<value>Flow field</value>
+	</string>
+	<string>
+		<key>Generator_FlowField.name</key>
```

Comment:
duplicate (to be deleted to pass the CI)
Note to myself : set as trace the expected errors (test of the test) so the real errors become obvious in the logs of the buid...

Reply:
yes please.  every time I am hunting for the value in the extensive error report.